CSS/CSS.sublime-syntax

```diff
@@ -271,6 +278,36 @@ contexts:
         - match: '\s*(?=[{;])'
           pop: true
         - include: media-query
+# Namespace At-Rule: https://www.w3.org/TR/css3-namespace/
+  namespace:
+    - match: \s*((@)namespace)\s*\b(\S*)
+      captures:
+        1: keyword.control.at-rule.namespace.css
+        2: punctuation.definition.keyword.css
+        3: entity.other.namespace-prefix.css
+      push:
+        - meta_scope: meta.at-rule.namespace.css
+        - match: '\s*(;|$)'
+          captures:
+            1: punctuation.terminator.rule.css
```

Comment:
I'd use a lookahead here: `(?=;|$)` so that we don't have to duplicate the scope for `;`.